### PR TITLE
Verify keys in bulk

### PR DESCRIPTION
**Pull request by F4ever**
### Before merging Pull Request to master make sure:
- [ ] You added tests to new feature
- [ ] Tests are green
- [ ] Version in setup.py incremented
- [ ] Version in pyproject.toml is same as in setup.py
- [ ] You added description with updates to CHANGELOG.md
- [ ] You updated README.md (if applicable)
- [ ] Lido SDK from [Test PyPI](https://test.pypi.org/project/lido-sdk/) is working well  (lib updates after each comment)

### CODE REVIEW

**Comment by F4ever** [comment]
@avsetsin Look here pls

File: lido_sdk/config.py [hunk 1]
```diff
@@ -3,4 +3,4 @@
 MULTICALL_MAX_WORKERS: int = 6
 MULTICALL_MAX_RETRIES: int = 5
 MULTICALL_POOL_EXECUTOR_TIMEOUT: int = 30
-VALIDATE_POOL_EXECUTOR_TIMEOUT: int = 10
+VALIDATE_POOL_EXECUTOR_TIMEOUT: int = 100
```
Comment (avsetsin) [inline lido_sdk/config.py:6]: is it in ms? after switching to batches of 1000 keys, will the script have enough to validate the keys?

Comment (avsetsin) [reply lido_sdk/config.py:6]: so it's in seconds, then it's fine

**Review by avsetsin — APPROVED** [review]
lgtm

**Pull request merged**